topics/epigenetics/tutorials/ewas-suite/tutorial.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: tutorial_hands_on
+topic_name: epigenetics
+tutorial_name: ewas-suite
+---
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+The field of cancer genomics has demonstrated the power of massively parallel sequencing techniques to inform on the genes and specific alterations that drive tumor onset and progression. Although large comprehensive sequence data sets continue to be made increasingly available, data analysis remains an ongoing challenge, particularly for laboratories lacking dedicated resources and bioinformatics expertise. To address this, we have provide training based on  Galaxy tools ewas suite that represent many popular algorithms for detecting somatic genetic alterations from cancer genome and exome data.  
+
+> ![epimechanism](../../images/Epigenetic_mechanisms.jpg)
+
+This exercise uses the dataset from the Cell publication by Hugo, Willy, et al., 2015 (DOI: 10.1016/j.cell.2015.07.061). The goal of this article was to identify differentially methylated regions and positions associated with treatment resistant melanomas.
+
+To this end, Illumina 450K Methylation array was performed in melanoma tumors from example patient pre and post MAPKi and BRAFi treatment with  different outcomes (sensitive and resistant). For each sample there is raw green and red colour arrays containing the summarized bead information generated bythe Illumina 450k scanner, i.e. there are 8 files in total:
+
+
+The Infinium Methylation Assay 450k uses two different bead types to detect changes in DNA methylation levels. In the figure we can see M - methylated and U - unmethylated bead types. In our study unmethylated and methylated bead signals are reported as green and red colors respectively.
+
+> ![methassay](../../images/methylationassay.png)
+
+
+Accession  | Sensitivity | Treatment
+--- | ---  | ---
+GSM1588704 | baseline    | pre-treatment
+GSM1588705 | baseline    | pre-treatment 
+GSM1588706 | resistant   | BRAFi
+GSM1588707 | resistant   | BRAFi
+
+> ### Agenda
+> ![ewaschart1](../../images/ewaschart1.png)
```

Comment:
Do you need the images there in the Agenda? You could explain globally the idea of the workflow before with the images as support